app/src/main/java/ru/chaichuk/hwapp/db/MoviesTypeConverters.kt

```diff
@@ -0,0 +1,39 @@
+package ru.chaichuk.hwapp.db
+
+import androidx.room.TypeConverter
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import ru.chaichuk.hwapp.data.Actor
+import ru.chaichuk.hwapp.data.Genre
+
+class MoviesTypeConverters {
+
+    @TypeConverter
+    fun fromGenres(genres : List<Genre>) : String {
+        val gson = Gson()
+        val type = object : TypeToken<List<Genre>>() {}.type
+        return gson.toJson(genres, type)
+    }
+
+    @TypeConverter
+    fun toGenres(genresJson : String) : List<Genre> {
```

Comment:
that's a bad idea
in the movie has only one genre 
and that you need to create a genre table and save genre id inside a movie

Reply:
I did not find any solution to store one-to-many in room framework without adding new entity classes and additional classes with relations. So I decided to make less code and store all that data as text in the same record. 
In next commit I will add that additional tables and data classes

Reply:
made new commit with separate tables for each entity